install/0000_90_cluster-version-operator_02_servicemonitor.yaml

```diff
@@ -75,6 +78,7 @@ spec:
         max by (name, condition, endpoint) (cluster_operator_conditions{name="version", condition="Upgradeable", endpoint="metrics"} == 0)
       for: 60m
       labels:
+        namespace: openshift-cluster-version
```

Comment:
Here's another case where I'd rather include `namespace` in the aggregation (`max by (namespace, name, condition, endpoint) (...)`), instead of clearing the label and then injecting a hard-coded replacement.

Reply:
Thank you, fixed in https://github.com/openshift/cluster-version-operator/pull/800/commits/d525eb97a2c971b4837210963d86d3b617452891.